daos/pgsql/Database.php

```diff
@@ -227,6 +227,12 @@ public function __construct() {
                         'INSERT INTO version (version) VALUES (12)'
                     ]);
                 }
+                if (strnatcmp($version, '13') < 0) {
+                    \F3::get('db')->exec([
+                        'ALTER TABLE ' . \F3::get('db_prefix') . 'items UNIQUE (uid (150), source);',
+                        'INSERT INTO ' . \F3::get('db_prefix') . 'version (version) VALUES (13)'
```

Comment:
There is no need to use `db_prefix` in Postgres. It is a measure for cheap shared hostings that only provide a single MySQL database.

Reply:
fix in 7040883bfbd248dae6c5d266912379c31cfcacd5